### ISSUE TITLE
Do not throw error when function has no parameter and returns

### ISSUE THREAD

**Issue by tohjg** [gajus/eslint-plugin-jsdoc#77]
Is there any configuration to set the subject mentioned above?

Naming convention of a function should be meaningful and descriptive. Thus, merely a description of the function is redundant in JSDoc comment. Too much comments make code hard to read.

eg.

```
renewSubmitButtonState() {
   ...
}
```
